packages/coordinator/src/screens/bookedDonation/InfoBar.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./InfoBar.module.scss";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+
+export type InfoBarProps = {
+  title: string;
+  value: React.ReactNode;
```

Comment:
tip: If you call this prop `children` it will be easier to use it.

Calling this element would look like
```
      <InfoBar
        title={"מספר טלפון"}
        icon={<Phone />}
      >
           <a href={`tel: ${props.appointment.phone}`}>
            {props.appointment.phone}
           </a>
      </InfoBar>
```

Instead of 
```
      <InfoBar
        title={"מספר טלפון"}
        value={
          <a href={`tel: ${props.appointment.phone}`}>
            {props.appointment.phone}
          </a>
        }
        icon={<Phone />}
      />
```